.travis.yml

```diff
@@ -8,45 +8,70 @@ matrix:
       env:
         - SPEEDUPS=1
         - NUMPY=1
+        - GEVENT=0
+        - GEVENT_PATCHED=0
```

Comment:
@snorfalorpagus I wanted to be able to test both gevent being present but `patch_all` not being invoked _and_ gevent being present and `patch_all` is invoked.